src/test/java/org/jfrog/hudson/pipeline/integration/CommonITestsPipeline.java

```diff
@@ -305,14 +306,15 @@ void gradleCiServerTest(String buildName) throws Exception {
     }
 
     void gradleCiServerPublicationTest(String buildName) throws Exception {
-        Set<String> expectedArtifacts = Sets.newHashSet(pipelineType.toString() + "-gradle-example-ci-server-publication-1.0.jar", pipelineType.toString() + "-gradle-example-ci-server-publication-1.0.pom", pipelineType.toString() + "-gradle-example-ci-server-publication-1.0.module");
+        Set<String> expectedArtifacts = Sets.newHashSet(pipelineType.toString() + "-gradle-example-ci-server-publication-1.0.jar", pipelineType.toString() + "-gradle-example-ci-server-publication-1.0.pom");
         String buildNumber = "3";
         try {
             runPipeline("gradleCiServerPublication", false);
             Build buildInfo = getBuildInfo(buildInfoClient, buildName, buildNumber);
             assertEquals(5, buildInfo.getModules().size());
 
             Module module = getAndAssertModule(buildInfo, "org.jfrog.example.gradle:" + pipelineType.toString() + "-gradle-example-ci-server-publication:1.0");
+            module.setArtifacts(module.getArtifacts().stream().filter(art -> !art.getType().toLowerCase().equals("module")).collect(Collectors.toList()));
```

Comment:
Please add a comment explaining that we remove the `.module` files to allow the test to run on >=6.